### PR TITLE
feat: add generic readme to templates/js

### LINKED ISSUE #63: README.md is missing from the generated client
**Describe the bug**
When I generate a new client. The folder containing the client should contain a README 

**To Reproduce**
1. `open-rpc-generator-client -s https://raw.githubusercontent.com/etclabscore/multi-geth/0718d25e92e568f486be17123e9384ce15d8b48c/openrpc.json EthClient`
2. `cd EthClient`
3. See no `README.md`

**Expected behavior**
I should see at least a generic `README.md` for a generated RPC client

**Screenshots**
image

### PR DESCRIPTION
**Pull request by stevanlohja**
Generic initial readme for template/js.
This closes #63

### CODE REVIEW

File: templates/js/static/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# OPEN-RPC Client
+
+Generated OpenRPC client.
+
+## Dependencies
+
+* [TypeScript](https://www.typescriptlang.org/)
+* _At least_ Node 8x.
+* [VSCode](https://code.visualstudio.com/) + [OPEN-RPC vscode extension](https://marketplace.visualstudio.com/items?itemName=OPEN-RPC.OPEN-RPC).
```
Comment (BelfordZ) [inline templates/js/static/README.md]: Vs code isnt a dep

File: templates/js/static/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# OPEN-RPC Client
+
+Generated OpenRPC client.
+
+## Dependencies
+
+* [TypeScript](https://www.typescriptlang.org/)
```
Comment (BelfordZ) [inline templates/js/static/README.md]: typescript is in node modules, not a dep that needs to be mentioned here

File: templates/js/static/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+After generating a template client `cd` into the client directory and `ls`.
+
+```
+build             docs              index.ts          node_modules      package-lock.json package.json      tsconfig.json     tslint.json
+```
+
+## Understanding the Structure
+
+* buidl/: the built or --out directory of the client.
```
Comment (BelfordZ) [inline templates/js/static/README.md]:
```suggestion
* build/: the built or --out directory of the client.
```

File: templates/js/static/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# OPEN-RPC Client
+
+Generated OpenRPC client.
+
```
Comment (BelfordZ) [inline templates/js/static/README.md]: first things first, should show example usage.

Comment (shanejonas) [reply templates/js/static/README.md]: yea maybe we can show the example usage from the output of the command in the root README to generate the client?

this one:
```
open-rpc-generator-client \
  -s https://raw.githubusercontent.com/open-rpc/examples/master/service-descriptions/petstore-openrpc.json \
  PetStore
```

File: templates/js/static/README.md (hunk 1 above)
Comment (stevanlohja) [inline templates/js/static/README.md]:
```suggestion
```

**Pull request closed without merging**